### ISSUE TITLE
Add a link to the Alerts page for a candidate from the scanning page

### ISSUE THREAD

**Issue by kmshin1397** [fritz-marshal/fritz#88]
Waiting on #87.
Should be straightforward once new extension infrastructure is in place.

**Comment by mcoughlin** [comment]
Closing as obsolete: concerns the Kowalski-era alert/filter stack, superseded by BOOM. Please refile in skyportal/BOOM if still relevant.
